repositories.bzl

```diff
@@ -0,0 +1,559 @@
+def boringssl_repositories(bind=True):
+    native.git_repository(
+        name = "boringssl",
+        commit = "bfd36df3da38dbf8828e712f42fbab2a0034bc40",  # 2017-02-02
+        remote = "https://boringssl.googlesource.com/boringssl",
+    )
+
+    if bind:
+        native.bind(
+            name = "boringssl_crypto",
+            actual = "@boringssl//:crypto",
+        )
+
+        native.bind(
+            name = "libssl",
+            actual = "@boringssl//:ssl",
+        )
+
+
+def protobuf_repositories(bind=True):
+    native.git_repository(
+        name = "protobuf_git",
```

Comment:
I know that I started this, but could we get rid of all those `_git` and `_tar` suffixes? They are just annoying if we're referring to external repositories by name and not using `//external:` bindings.

(applies to the whole file)